### ISSUE TITLE
Bug: Multiblocks leave placeholder behind

### ISSUE THREAD

**Issue by Lejo1** [mt-mods/jumpdrive#71]
When you jump only the bottom of a door (or other Multiblocks like the ones from digiline_routing) the placeholder at the top will left behind.
After you jumped you can place blocks into the top of the door.
And nobody can destroy the placeholder.

**Comment by BuckarooBanzay** [comment]
This is a known problem (did you see `lga`'s "museum of oddities on pandorabox :smile:).
Possible solutions:
* Plugin api to register multinode blocks (and then block jump or remove them)
* ...idk, i'm open for ideas

**Comment by Lejo1** [comment]
Hmm,
I'm not sure what's the best way to fix it.

Just wanted to report the bug :)

**Comment by S-S-X** [comment]
I don't think disassembling things with jumpdrive is bad thing at all, it is logical that some things can break apart.

However there's also those things that use invisible placeholders like said digiline routing nodes, these should be cleaned up or blacklisted.
Digiline routing components also have actual digiline functionality on placeholder node, these nodes are invisible but digilines will connect to these nodes. Also not too easy to remove.

Blacklisting such components completely does not sound like nice idea but blacklisting around jump area edges could make sense while still simple to actually implement.

In my opinion as more general, but also probably close to impossible, solution for many nodes best fix would be to remove those placeholder nodes completely from mod, those placeholders just limit creativity on game that asks you to be creative.

**Comment by OgelGames** [comment]
> I don't think disassembling things with jumpdrive is bad thing at all, it is logical that some things can break apart.

I agree, especially for things that look like they are multiple nodes, such as fancy vendors.

> blacklisting around jump area edges

That would probably be the most flexible solution 👍 


> And nobody can destroy the placeholder.

That's what technic constructors are for 😄
